src/build_workflow/build_args.py

```diff
@@ -52,6 +60,8 @@ def __init__(self):
         self.snapshot = args.snapshot
         self.component = args.component
         self.keep = args.keep
+        self.platform = args.platform
+        self.arch = args.arch
```

Comment:
You need defaults for these or it will be blank and break the existing OpenSearch build.